lib/usd/ui/layerEditor/layerTreeItem.cpp

```diff
@@ -282,15 +281,21 @@ bool LayerTreeItem::isLocked() const { return _layer && _layer->PermissionToEdit
 
 bool LayerTreeItem::appearsLocked() const
```

Comment:
Named similar to appearsMuted(). It has to do with the parent / children's state of lock.
By design, some of the sublayer's actions cannot be performed if the parent is locked.